test/cli/package-autocorrect-missing-import/test.out

```diff
@@ -67,7 +67,7 @@ use_other_package/foo.test.rb:4: `Test::Foo::Bar::OtherPackage::TestUtil` resolv
      3 |class Foo::Bar::OtherPackage < PackageSpec
```

Comment:
Can we add a test case where the autocorrect added is a test helper import, to test the logic for detecting which type of import we suggest an autocorrect for?

Just adding to `use_other_package` is fine, we don't need to test it for all the strict deps configs.